src/lib/openapi/spec/admin-features-query-schema.ts

```diff
@@ -0,0 +1,26 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const adminFeaturesQuerySchema = {
+    $id: '#/components/schemas/adminFeaturesQuerySchema',
+    type: 'object',
+    additionalProperties: true,
+    properties: {
+        tag: {
+            type: 'array',
+            items: {
+                type: 'array',
+                items: {
+                    type: 'string',
+                },
```

Comment:
Just to be clear, is it a list of list of tags? A `Tag[][]`? or is there an extra layer of nesting here?

If it is a list of lists, it definitely needs some extra explanation.

Reply:
It used to be a list of lists in the old version. Maybe we can dumb it down to simple an array of strings. Let me see how my tests run.